aea/cli/core.py

```diff
@@ -193,6 +197,100 @@ def add_key(ctx: Context, type_, file):
     )
 
 
+@cli.command()
+@click.argument(
+    "type_",
+    metavar="TYPE",
+    type=click.Choice(
+        [DefaultCrypto.identifier, FetchAICrypto.identifier, EthereumCrypto.identifier]
+    ),
+    required=True,
+)
+@pass_ctx
+def get_address(ctx: Context, type_):
+    """Get the address associated with the private key."""
+    try_to_load_agent_config(ctx)
+
+    _verify_or_create_private_keys(ctx)
+    private_key_paths = dict(
+        [
+            (identifier, config.path)
+            for identifier, config in ctx.agent_config.private_key_paths.read_all()
+        ]
+    )
+    try:
+        wallet = Wallet(private_key_paths)
+        address = wallet.addresses[type_]
+        logger.info("Your address is: {}".format(address))
```

Comment:
I propose to remove the prefix `"Your address is:"`. It is useful to have only the address string as output, e.g. for piping of commands etc.

Reply:
or, having an optional flag to implement my feature.

Reply:
Good point, let's just print the address